src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala

```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
     }
   }
 
+  def ExampleIssues: Unit = {
```

Comment:
INFO: Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))